scripts/coverage-report.js

```diff
@@ -38,20 +36,23 @@ async function artifactDir(repoSpec, workflowId, artifactName) {
         })).data.artifacts.find((a) => a.name === artifactName);
         let artifactRaw;
         try {
+            process.stdout.write(`Downloading ${artifactName}... `);
             artifactRaw = Buffer.from((await octokit.rest.actions.downloadArtifact({
                 owner, repo,
                 artifact_id: artifactData.id,
                 archive_format: "zip"
             })).data);
+            console.log(`${artifactRaw.byteLength.toLocaleString()} bytes received`);
         } catch (error) {
+            console.log(error.message);
             if (error.status === 410) {
                 return;  // Ignore error if artifact has expired
             } else {
                 throw error;
             }
         }
         tempDir = fs.mkdtempSync(owner);
-        await promisify(pipeline)(Readable.from(artifactRaw), unzipStream.Extract({ path: tempDir }));
```

Comment:
Would we be able to use [`zlib.unzip`](https://nodejs.org/docs/latest-v18.x/api/zlib.html#zlibunzipbuffer-options-callback) here? `zlib` is a part of Node.js core, so we could pass the buffer to that function and be able to remove the `unzip`-related dependencies 🙂